packages/wrangler/src/create-worker-upload-form.ts

```diff
@@ -156,7 +176,12 @@ export function createWorkerUploadForm(worker: CfWorkerInit): FormData {
         const name = module.name.replace(/[^a-zA-Z0-9_$]/g, "_");
         metadataBindings.push({
           name,
-          type: module.type === "compiled-wasm" ? "wasm_module" : "text_blob",
+          type:
+            module.type === "compiled-wasm"
```

Comment:
I should just make module.type have the same values as binding.type, so we don't have to do this kind of check when converting. Next PR.